pages/airdrop/claim.vue

```diff
@@ -161,8 +159,12 @@ export default class AirdropClaimPage extends Vue {
     return `${BIG_DIPPER_TX_BASE_URL}${this.currentMission.txHash}`
   }
 
-  mounted() {
-    this.fetchMissionDecay()
+  get isAirdropEnd() {
+    return this.decay.factor === 1;
```

Comment:
```suggestion
    return this.decay.factor >= 1;
```
In case the value somehow > 1